packages/css/src/logo/README.md

```diff
@@ -0,0 +1,35 @@
+# Logo
+
+Het logo van de gemeente bestaat uit een beeldmerk (3 Andreaskruisen) en een woordmerk (Gemeente Amsterdam).
+Het logo heeft een vaste kleur, grootte en positie en staat op al onze websites.
+
+## Specificaties
+
+Gebruik altijd het online logo.
+Het Nederlandse logo voor Nederlandstalige websites en anderstalige websites behalve Engels.
+Het Engelstalige logo gebruiken we voor Engelstalige websites.
+
+## Uitzondering
+
+Gebruik het submerklogo als de website eigendom is van 1 van de 4 submerken.
+Zij hebben een aparte status omdat er sprake is van een bijzondere vorm van dienstverlening die niet direct met de gemeente in verband wordt gebracht.
+
+De submerken zijn:
+
+- GGD Amsterdam
+- Stadsarchief
+- Stadsbank van Lening
+- VGA Verzekeringen
+
+## Eisen voor alle logo´s
+
+- Het logo wordt altijd getoond.
+- Het logo is linksboven uitgelijnd in de header.
+- Het logo verwijst naar de homepage.
```

Comment:
I know this causes confusion sometimes. The logo should be part of a link to the homepage of the relevant website, not always amsterdam.nl, right?

Reply:
Not sure what you’re suggesting. To specify ‘the homepage’ further?

Reply:
Yeah, something like this:

```suggestion
- Het logo verwijst naar de homepage van je website of applicatie.
```